src/Altinn.Apps/AppTemplates/AspNet/Altinn.App.PlatformServices/Interface/IAltinnApp.cs

```diff
@@ -97,6 +97,22 @@ public interface IAltinnApp
         /// <param name="data">The data to perform calculations on</param>
         Task<bool> RunCalculation(object data);
```

Comment:
We can mark this method as deprecated with the ObsoleteAttribute.

Reply:
done